overlay.yml

```diff
@@ -21,5 +21,10 @@ components:
   - src: github:kubernetes-incubator/cri-o
     override-version: "1.11"
 
-#   - src: github:coreos/ignition
-#   TODO: use copr for distgit here OR get spec in the ignition repo
+  - src: github:coreos/ignition
+    override-version: "0.24.0"
```

Comment:
We shouldn't need this, right? Looks like the tag is on the master branch. Would be good to keep those overrides to a minimum to keep maintenance low.

Reply:
```
$ git describe --tags --always --abbrev=42
v0.24.0-18-g042a00e218eb4a49eb22f053077c4e3152c8caed
```

Reply:
I'll double check but you're likely correct.

Reply:
You are correct. I've removed it.